webapp/channels/src/components/admin_console/color_setting.tsx

```diff
@@ -16,27 +16,27 @@ type Props = {
     disabled?: boolean;
 }
 
-export default class ColorSetting extends React.PureComponent<Props> {
-    private handleChange = (color: string) => {
-        if (this.props.onChange) {
-            this.props.onChange(this.props.id, color);
+const ColorSetting = (props: Props) => {
```

Comment:
nit: There is some tendency now to use the deconstruction here to ensure we are using everything that we have in the Props. Something like:
```suggestion
const ColorSetting = ({id, label, helpText, value, onChange, disabled}: Props) => {
```

That way then the rest of the code you don't need references to props, and in fact, if there is something that you don't use suddently the IDE makes it super obvious.

Reply:
Oops, looks like I already provided this feedback.

Reply:
Thanks for the feedback ☺️